electron/main.ts

```diff
@@ -144,6 +155,14 @@ const openApplication = async (givenPath?: string) => {
   ElectronStore.initRenderer();
   const win = createWindow();
 
+  const missingDependecies = APP_DEPENDENCIES.filter(d => !which(d));
+
+  if (missingDependecies.length > 0) {
+    win.webContents.on('did-finish-load', () => {
+      win.webContents.send('missing-dependecy-result', {dependencies: missingDependecies});
```

Comment:
typo in "missing-dependecy-result" - should be "missing-dependency-result" in all places...